api/src/identity-access-management/application/oidc-provider/oidc-provider.route.js

```diff
@@ -7,13 +7,14 @@ export const oidcProviderRoutes = [
   {
     method: 'GET',
     path: '/api/oidc/identity-providers',
-    config: {
+    options: {
```

Comment:
Ce renommage ne correspond pas au titre du commit.

Il serait préférable : 
* soit de ne pas faire cette modification du tout
* soit de faire cette modification dans un autre commit et d'expliquer pourquoi

Reply:
Depuis Hapi.dev v17, config a été déprécié et remplacé par options. Toutefois le nom config restait pour compatibilité ascendante.
Or lors de nos tests, il semble que config.cache ne soit pas pris en compte, d'où le changement vers la notation state of the art afin de pouvoir changer le comportement du cache.

Reply:
Bien reçu. Donc faire un commit dédié et qui explique de manière minimaliste pourquoi STP. Et mettre l'explication que tu viens d'écrire dans la description de la PR STP :pray: